railties/lib/rails/generators/rails/app/templates/test/test_helper.rb.tt

```diff
@@ -5,9 +5,9 @@ require 'rails/test_help'
 class ActiveSupport::TestCase
   # Run tests in parallel with specified workers
 <% if defined?(JRUBY_VERSION) || Gem.win_platform? -%>
-  parallelize(workers: 2, with: :threads)
+  parallelize(workers: :number_of_processors, with: :threads)
 <%- else -%>
-  parallelize(workers: 2)
+  parallelize(workers: :number_of_processors)
```

Comment:
If we're making the default an "it just works" value like this, does it make sense to still include it here? If most people don't need to override it, maybe this line can just be a plain `parallelize`?

Reply:
I thought that it would be better to keep this in this form since it would be easier for users to figure out that their tests use the actual core count as a number of workes. But I don't have strong opinions. I think that we should either simplify this to `parallelize` and `parallelize(with: :threads)`(a few lines above), or extend it to `parallelize(workers: :number_of_processors, with: :processes)` to emphasize that test suit run use "fork approach".